### ISSUE TITLE
passing information from child class to parent, using dynamic classes in kvlang (Kivy)

### ISSUE THREAD

**Issue by yoelk** [kivy/kivy#3186]
For more details, please look in the stackoverflow question:
http://stackoverflow.com/questions/28684165/passing-information-from-child-class-to-parent-using-dynamic-classes-in-kvlang

**Comment by dessant** [comment]
I don't think this is a bug. I see you've already managed to fix your original issue, and passing the label object from the child button to the parent tree does not make much sense given that the label id is already in the parent, ready to reference.

As a sidenote, please describe you issue here on github in the future and provide the relevant code in this place.

**Comment by yoelk** [comment]
Got it. I'm just used to stackoverflow so I thought it would be the
platform for such questions.
But I might have not explained the problem clearly enough, as the problem
remains:

What I want to do (which I thought would be straightforward) is to create a
class (Button1) that holds an ObjectProperty. And I want that this property
will be initialized by instances of Button1, and this doesn't seem to work.
Obviously, the label/button example was just to simplify things, I'm aiming
at a more elaborate case with many such object properties. I just wanted to
see how to make the simplest case work.

What puzzles me is that if I use a StringProperty instead of an
ObjectProperty, I can have the instance of Button1 to use it.

To summarize, I'm trying to find the kv way of passing arguments to the
__init__ function of the parent class.
Can you help me do that?
